.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
It might be worth not assuming everybody is using flakes:
```suggestion
if nix flake info &>/dev/null; then
  use flake
else
  use nix
fi
```

Reply:
+1